styles/header.css

```diff
@@ -0,0 +1,109 @@
+html {
+    font-family: sans-serif;
+    font-size: 14px;
+    color: #2c2b25;
+}
+
+body {
+    width: 100%;
+    margin: 0;
+}
+
+.top-nav {
+    display: flex;
+    flex-direction: row;
+    padding: 2.5vw;
+}
+
+.logo {
+    width: 100%;
+    height: auto;
+    max-width: 18vw;
+}
+
+.search-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    width: 100%;
+}
+
+.search-box {
+    padding: 0;
+    width: 29vw;
+    height: 3.3em;
```

Comment:
Try to use virtual measure everywhere if you're starting using it in other places. Stick to 2-3 instead of using all of them (rem, em, vh, vw, %, px etc)